charts/images.yaml

```diff
@@ -271,14 +271,32 @@ images:
   sourceRepository: github.com/kubernetes/autoscaler
   repository: registry.k8s.io/autoscaling/vpa-admission-controller
   tag: "0.11.0"
+  targetVersion: "< 1.21"
+- name: vpa-admission-controller
+  sourceRepository: github.com/kubernetes/autoscaler
+  repository: registry.k8s.io/autoscaling/vpa-admission-controller
+  tag: "0.12.0"
+  targetVersion: ">= 1.21"
```

Comment:
The `FindImage` calls in

https://github.com/gardener/gardener/blob/9bec5d99aca719ac2a3cd7961e9df9f1971cfa7e/pkg/operation/seed/components.go#L379-L397

have to be updated to specify the `imagevector.TargetVersion(seedVersion.String())` option. Otherwise it picks up VPA 0.11.0 for K8s >= 1.21 Seed cluster.